pom.xml

```diff
@@ -61,6 +61,7 @@
     	<module>binding</module>
 		<module>features</module>
 		<module>products</module>
+		<module>protocols/enocean</module>
```

Comment:
I would actually not add it to the main reactor build, but rather treat it as a sub-project that can be build independently of the ESH bundles. Especially as the "mvn clean" + "mvn install" trick will otherwise frustrate ESH developers that are not interested in the EnOcean Base driver. Once merged, I will setup a dedicated Hudson build job (on hudson.eclipse.org/smarthome/) to build these artifacts.